### ISSUE TITLE
Multi Files

### ISSUE THREAD

**Issue by nseb** [carsten-klaffke/send-intent#70]
Hello,
is it possible to receive several files like images? , when testing with Android, the application disappears from the sharing choice.
I also saw that there was an open ticket also for IOS.

Best Regards

**Comment by carsten-klaffke** [comment]
Yes, you can share multiple files. I just created an Example application (/Example/SendIntentExample). There you can see how to configure AndroidManifest.xml for sharing multiple files.
